app/models.py

```diff
@@ -466,6 +466,40 @@ def serialize(self):
         }
 
 
+class ServiceCallbackApi(db.Model, Versioned):
+    __tablename__ = 'service_callback_api'
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    service_id = db.Column(UUID(as_uuid=True), db.ForeignKey('services.id'), index=True, nullable=False, unique=True)
+    service = db.relationship('Service', backref='service_callback_api')
+    url = db.Column(db.String(), nullable=False)
+    _bearer_token = db.Column("bearer_token", db.String(), nullable=False)
+    created_at = db.Column(db.DateTime, default=datetime.datetime.utcnow, nullable=False)
+    updated_at = db.Column(db.DateTime, nullable=True)
+    updated_by = db.relationship('User')
+    updated_by_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), index=True, nullable=False)
+
+    @property
+    def bearer_token(self):
+        if self._bearer_token:
+            return encryption.decrypt(self._bearer_token)
+        return None
+
+    @bearer_token.setter
+    def bearer_token(self, bearer_token):
+        if bearer_token:
+            self._bearer_token = encryption.encrypt(str(bearer_token))
```

Comment:
I don't think this should be encrypted in the database - we need to read it out in plain text anyway to add it to the outbound message

so i think we can just have this as a normal field rather than a property

Reply:
It is not right to store someone's secret in the database in plain text. We should mask this. We can always change this if we go for a more secure algorithm in the future.

Reply:
This is the same technique used for api keys, bearer tokens for the inbound api, and personalisation.

Reply:
After discussion, we will keep the coherence of the database encode method and refractor the 'encrypt' 'decrypt' keywords in a separate PR.